src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
Yeah, I just stated my personal taste, I can agree to have these operators in new code. But probably it will be quite a long time until all ! && and || are replaced.